minecraft_servers/claymc/manifest.json

```diff
@@ -0,0 +1,31 @@
+{
+  "server_name": "claymc",
+  "nice_name": "ClayMC",
+  "direct_ip": "play.claymc.net",
+  "server_wildcards": [
+    "%.claymc.net"
```

Comment:
Doesn't look like a real wildcard
<img width="904" alt="image" src="https://github.com/LabyMod/server-media/assets/37186532/1902041f-d0ff-4598-93fc-4888140e964f">

Reply:
As I already said. There's a A-Record on the ClayMC.net Domain

Reply:
But thats not a **wilcard** record: https://en.wikipedia.org/wiki/Wildcard_DNS_record

Reply:
Fixed

Reply:
Looks good! 👍
<img width="931" alt="image" src="https://github.com/LabyMod/server-media/assets/37186532/47f6762f-b38d-45c6-b77b-e31ac47979ca">